pkg/agent/route/route_linux.go

```diff
@@ -1163,21 +1163,42 @@ func (c *Client) AddClusterIPRoute(svcIP net.IP) error {
 			} else {
 				c.clusterIPv4CIDR = newClusterIPCIDR
 			}
-			klog.V(4).InfoS("Created a route with new ClusterIP CIDR to route the ClusterIP to Antrea gateway", "ClusterIP CIDR", newClusterIPCIDR, "clusterIP", svcIP)
+			klog.V(4).InfoS("Created a route with new ClusterIP CIDR to route the ClusterIP to Antrea gateway", "ClusterIP CIDR", newClusterIPCIDR, "ClusterIP", svcIP)
 		} else {
-			klog.V(4).InfoS("Route with current ClusterIP CIDR can route the ClusterIP to Antrea gateway", "ClusterIP CIDR", curClusterIPCIDR, "clusterIP", svcIP)
+			klog.V(4).InfoS("Route with current ClusterIP CIDR can route the ClusterIP to Antrea gateway", "ClusterIP CIDR", curClusterIPCIDR, "ClusterIP", svcIP)
 		}
 	} else {
+		// Create the first ClusterIP route on Antrea gateway.
 		route := generateRoute(svcIP, mask, gw, linkIndex, scope)
 		if err := netlink.RouteReplace(route); err != nil {
 			return fmt.Errorf("failed to install new ClusterIP route: %w", err)
 		}
-
+		klog.V(4).InfoS("Created a route to route the ClusterIP to Antrea gateway", "ClusterIP", svcIP)
 		if isIPv6 {
 			c.clusterIPv6CIDR = route.Dst
 		} else {
 			c.clusterIPv4CIDR = route.Dst
 		}
+
+		// List all existing routes on Antrea gateway.
+		routes, err := c.listIPRoutesOnGW()
+		if err != nil {
+			return fmt.Errorf("error listing ip routes: %v", err)
```

Comment:
nit: you use `%v` here to format errors but `%w` below, why the inconsistency?

Reply:
Thanks for reminding.